src/constants/variables.ts

```diff
@@ -1,8 +1,14 @@
 export class Variables {
-  static FUSE_CHAIN_ID = 122;
-  static FUSE_RPC_URL = 'https://rpc.fuse.io';
-  static DEFAULT_GAS_LIMIT = 700000;
-  static NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'; // For sending native (ETH/FUSE) tokens
-  static BASE_URL = 'api.fuse.io';
-  static SOCKET_SERVER_URL = 'wss://ws.fuse.io/connection/websocket';
+  static FUSE_CHAIN_ID = process.env.FUSE_CHAIN_ID ?? 122;
+  static FUSE_RPC_URL = process.env.FUSE_RPC_URL ?? 'https://rpc.fuse.io';
+  static DEFAULT_GAS_LIMIT = process.env.DEFAULT_GAS_LIMIT ?? 700000;
+  static NATIVE_TOKEN_ADDRESS = process.env.NATIVE_TOKEN_ADDRESS ?? '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'; // For sending native (ETH/FUSE) tokens
```

Comment:
Ok this one is a const so it's ok to be static as 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE

Reply:
@leonprou reverted